packages/realm/src/GeoSpatial.ts

```diff
@@ -0,0 +1,72 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2023 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+export type IGeoPosition = [number, number]; //long/lat, so it's the same order as geoJSON
+
+//Interface that satisfies the geoJSON specification for a polygon.
+export interface IGeoPolygon {
```

Comment:
Agree with the naming.
Regarding exporting, I think we should at least export `IGeoPoint`, because developers are expected to make their objects conform to this interface in order for queries to work. 
Regarding `IGeoPolygon`, I'm not sure if that's necessary, but probably we could keep it for completeness.